static/src/javascripts/projects/common/modules/commercial/contributions-google-docs.js

```diff
@@ -4,13 +4,18 @@ import { getEpicParams } from 'common/modules/commercial/acquisitions-copy';
 
 const epicGoogleDocUrl: string =
     'https://interactive.guim.co.uk/docsdata/1fy0JolB1bf1IEFLHGHfUYWx-niad7vR9K954OpTOvjE.json';
+const bannerGoogleDocUrl: string =
+    'https://interactive.guim.co.uk/docsdata/1CIHCoe87hyPHosXx1pYeVUoohvmIqh9cC_kNlV-CMHQ.json';
 
 const getGoogleDoc = (url: string): Promise<any> =>
     fetchJSON(url, {
         mode: 'cors',
     });
 
 export const getEpicGoogleDoc: Promise<any> = getGoogleDoc(epicGoogleDocUrl);
+export const getBannerGoogleDoc: Promise<any> = getGoogleDoc(
```

Comment:
do we want an engagement banner version of `googleDocEpicControl()`

Reply:
Currently the control in the google doc isn't being used, it's using the hard coded version. Will update this!